apps/frontend/src/pages/profile/styles.module.css

```diff
@@ -1,16 +1,18 @@
-.title {
-	margin-top: 0;
-	margin-bottom: 24px;
-}
-
-.profile-loader {
+.profile-layout {
 	display: flex;
-	flex: 1;
-	align-items: center;
-}
-
-.profile-content {
+	flex-direction: column;
+	gap: 24px;
+	align-items: flex-start;
 	width: 100%;
 	max-width: 800px;
+	padding: 0;
 	margin: 0 auto;
+	color: var(--color-text-primary);
+}
+
+.title {
+	margin: 0;
+	font-size: 30px;
+	line-height: 120%;
+	word-break: break-word;
```

Comment:
We need this only for user input values, like project title etc.
```suggestion
```

Reply:
oh, got it